web-app/js/portal/cart/DownloadEstimator.js

```diff
@@ -137,9 +137,9 @@ Portal.cart.DownloadEstimator = Ext.extend(Object, {
         var params = filters.filter(function(filter) {
             return filter.isNcwmsParams;
         })[0];
-        var bounds = collection.getBounds();
 
-        if (bounds && params && params.latitudeRangeStart != undefined) {
```

Comment:
What if params is null?
`if (bounds && params && params.latitudeRangeStart != undefined) {` 
is safer